trello/1.0.0/README.md

```diff
@@ -0,0 +1,43 @@
+  # Keptn Trello Integration
+
+  ## Step 1: Gather Trello Information
+
+  1. Go to `https://trello.com/app-key` and make a note of your api-key
+  2. Click on the `Token` link and follow the wizard. Then make a note of your `api-token`
+  3. Make a note of the `board id` from the URL
+  4. Make a note of the (case sensitive) list name on your board. (eg. `To Do`)
+  5. Execute a curl request to get the IDs for the lists. This is the list under which you want the cards to be created
+  ```
+  curl -X GET https://api.trello.com/1/boards/<BOARD_ID>/lists?key=<TRELLO_API_KEY>&token=<TRELLO_API_TOKEN>
+  ```
+  The output should look like this. Make a note of the `id` field (eg. `12342044e670786dbbd21234`)
+  ```
+  [{
+    "id": "12342044e670786dbbd21234",
+    "name": "To Do",
+    "closed": false,
+    "idBoard": "12342044e670786dbbd21234",
+    "pos": 16384,
+    "subscribed": false,
+    "softLimit": null
+  }]
+  ```
+
+  ## Step 2: Save as Keptn Secret
+  Save the above details as a Keptn secret with the scope set to `keptn-webhook-service`
+
+  ![keptn secret](assets/1.png)
```

Comment:
As described in https://github.com/keptn-contrib/artifacthub/pull/104 Artifacthub currently doesn't support relative paths. The reason is that when processing packages AH only reads the metadata files. They don't store or serve any other files. Therefore all the relative paths in the PR should be changed to absolute paths. Will also add this fact to the README to avoid future problems.